### ISSUE TITLE
Deploy goal fails. Probably missing a primary artifact

### ISSUE THREAD

**Issue by hugithordarson** [undur/vermilingua-maven-plugin#7]
To make the plugin play nice with other plugins, having it define a primary artifact might be a capital idea

**Comment by paulhoadley** [comment]
This one is interesting. For reference, `wolifecycle` will produce:

```
Juno.woapplication.tar.gz
Juno.wowebserverresources.tar.gz
```

and _both of those_ get shipped to your Maven repo on `deploy`. We're going to need to replicate something similar, aren't we—with or without the WSR part, though you've just shipped the `performSplit` option! (It might be a conversation for another ticket, but is "split install" a deployment anachronism? I mean, _we currently use it,_ but as discussed elsewhere, there's no real need to when you're running appserver and webserver on the same box. I suppose the residual use case is where you're not.)

In any case, we need _some way_ to ship the WOA bundle, don't we?

**Comment by hugithordarson** [comment]
Yeah, this one probably needs solving, for exactly the reasons you mention.
